data-prepper-plugins/mutate-string-processors/src/main/java/com/amazon/dataprepper/plugins/processor/mutatestring/SplitStringProcessorConfig.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.amazon.dataprepper.plugins.processor.mutatestring;
+
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.NotNull;
+
+import java.util.List;
+
+public class SplitStringProcessorConfig {
+    public static class Entry {
```

Comment:
Are we supporting a configuration option to limit the number of times the string is split or was that feature deferred?

Reply:
That was deferred as I couldn't see the great usefulness it would bring to the table for the customer, but it can be added in fairly easily and quickly I believe.

Reply:
Based on the RFC discussion, it is a future item.